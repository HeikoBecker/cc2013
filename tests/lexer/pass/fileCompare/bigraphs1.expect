bigraphs1.c:1:1: keyword int
bigraphs1.c:1:5: identifier main
bigraphs1.c:1:9: punctuator (
bigraphs1.c:1:10: keyword int
bigraphs1.c:1:14: identifier argc
bigraphs1.c:1:18: punctuator ,
bigraphs1.c:1:20: keyword char
bigraphs1.c:1:24: punctuator *
bigraphs1.c:1:26: identifier argv
bigraphs1.c:1:30: punctuator <:
bigraphs1.c:1:32: punctuator :>
bigraphs1.c:1:34: punctuator )
bigraphs1.c:1:36: punctuator <%
bigraphs1.c:2:2: keyword return
bigraphs1.c:2:9: constant 0
bigraphs1.c:2:10: punctuator ;
bigraphs1.c:3:1: punctuator %>
