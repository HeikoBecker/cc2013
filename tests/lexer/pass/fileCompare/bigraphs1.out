/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:1: keyword int
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:5: identifier main
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:9: punctuator (
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:10: keyword int
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:14: identifier argc
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:18: punctuator ,
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:20: keyword char
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:24: punctuator *
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:26: identifier argv
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:30: punctuator <:
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:32: punctuator :>
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:34: punctuator )
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:1:36: punctuator <%
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:2:2: keyword return
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:2:9: constant 0
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:2:10: punctuator ;
/Users/pklitzke/c4/tests/lexer/pass/fileCompare/bigraphs1.in:3:1: punctuator %>
